end-end-tests/api-standards/test-bulk.bash

```diff
@@ -14,15 +14,17 @@ trap "rm -rf $tempdir" EXIT
 set +e
 
 function assert_ok {
-    if [ "$?" != "0" ]; then
-        echo "unexpected rc=$?"
```

Comment:
This `$?` was wrong, it was the status of the test above! That didn't affect the decision logic, but the log output was very wrong.